Makefile

```diff
@@ -21,11 +21,11 @@ endif
 include $(BOLOS_SDK)/Makefile.defines
 
 APPNAME = Ark
-APP_LOAD_PARAMS=--appFlags 0x40 --curve secp256k1 $(COMMON_LOAD_PARAMS)
+APP_LOAD_PARAMS=--appFlags 0x40 --curve secp256k1 --path "44'/111'" --path "44'/1'" $(COMMON_LOAD_PARAMS)
 
 APPVERSION_M=0
 APPVERSION_N=1
-APPVERSION_P=0
+APPVERSION_P=2
```

Comment:
i think it should be 4, see my general comment

Reply:
In https://github.com/ArkEcosystem/ledger/pull/15/files#diff-b67911656ef5d18c4ae36cb6741b7965L28 we change the version from 0.1.4 to 0.1.5.